### PR TITLE
Updated global surface mooring deploy csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updates per metadata review - so far just for GA01SUMO_Deploy and GS01SUMO_Deploy csvs.  Update - added GI01SUMO_Deploy csv updates.

### CODE REVIEW

**Comment by snwhite** [comment]
[Critical Metadata Revew [DT-15]](https://trello.com/c/OuquTzsl/15-critical-metadata-revew-dt-15)

File: deployment/GS01SUMO_Deploy.csv [hunk 1]
```diff
@@ -1,57 +1,57 @@
...
+AT-26-29,,NBP-15-11,,GS01SUMO-SBD12-01-OPTAAD000,1,1,2015-02-18T21:06:00,2015-12-27T11:20:00,CGMGS-01SUMO-00001,,CGINS-OPTAAD-00205,-54.4082,-89.3576,,1,4611,
+AT-26-29,,NBP-15-11,,GS01SUMO-SBD12-02-FLORTD000,1,1,2015-02-18T21:06:00,2015-12-27T11:20:00,CGMGS-01SUMO-00001,,CGINS-FLORTD-01212,-54.4082,-89.3576,,1,4611,
+AT-26-29,,NBP-15-11,,GS01SUMO-SBD12-03-HYDGN0000,1,1,2015-02-18T21:06:00,2015-12-27T11:20:00,CGMGS-01SUMO-00001,,CGINS-HYDGN0-99999,-54.4082,-89.3576,,0,4611,
+AT-26-29,,NBP-15-11,,GS01SUMO-SBD12-04-PCO2AA000,1,1,2015-02-18T21:06:00,2015-12-27T11:20:00,CGMGS-01SUMO-00001,,CGINS-PCO2AA-33182,-54.4082,-89.3576,,0,4611,
+AT-26-29,,NBP-15-11,,GS01SUMO-SBD12-05-WAVSSA000,1,1,2015-02-18T21:06:00,2015-12-27T11:20:00,CGMGS-01SUMO-00001,,CGINS-WAVSSA-05971,-54.4082,-89.3576,,0,4611,
+AT-26-29,,NBP-15-11,,GS01SUMO-SBD12-06-METBKA000,1,1,2015-02-18T21:06:00,2015-12-27T11:20:00,CGMGS-01SUMO-00001,,CGINS-METLGR-00017,-54.4082,-89.3576,,0,4611,
+AT-26-29,,NBP-15-11,,GS01SUMO-SBD12-08-FDCHPA000,1,1,2015-02-18T21:06:00,2015-12-27T11:20:00,CGMGS-01SUMO-00001,,CGINS-FDCHPA-43401,-54.4082,-89.3576,,-6,4611,
```
Comment (snwhite) [inline deployment/GS01SUMO_Deploy.csv:53]: Deployment 1 looks good.

File: deployment/GS01SUMO_Deploy.csv [hunk 2]
```diff
@@ -85,73 +85,73 @@ NBP-15-11,,NBP16-10,,GS01SUMO-RII11-02-PCO2WC052,2,1,2015-12-14T20:20:00,2016-12
...
 NBP-15-11,,NBP16-10,,GS01SUMO-SBD12-03-HYDGN0000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-HYDGN0-42356,-54.40408,-89.20691,,0,4588,
 NBP-15-11,,NBP16-10,,GS01SUMO-SBD12-04-PCO2AA000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-PCO2AA-31124,-54.40408,-89.20691,,0,4588,
 NBP-15-11,,NBP16-10,,GS01SUMO-SBD12-05-WAVSSA000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-WAVSSA-04571,-54.40408,-89.20691,,0,4588,
-NBP-15-11,,NBP16-10,,GS01SUMO-SBD12-06-METBKA000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-METLGR-00026,-54.40408,-89.20691,,-5,4588,
-NBP-15-11,,NBP16-10,,GS01SUMO-SBD12-08-FDCHPA000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-FDCHPA-44901,-54.40408,-89.20691,,-5,4588,
+NBP-15-11,,NBP16-10,,GS01SUMO-SBD12-06-METBKA000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-METLGR-00026,-54.40408,-89.20691,,0,4588,
+NBP-15-11,,NBP16-10,,GS01SUMO-SBD12-08-FDCHPA000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-FDCHPA-44901,-54.40408,-89.20691,,-6,4588,
```
Comment (snwhite) [inline deployment/GS01SUMO_Deploy.csv:103]: Deployment 2 ADCP depth should be 500 (not 501).

File: deployment/GS01SUMO_Deploy.csv [hunk 3]
```diff
@@ -85,73 +85,73 @@ NBP-15-11,,NBP16-10,,GS01SUMO-RII11-02-PCO2WC052,2,1,2015-12-14T20:20:00,2016-12
...
 NBP16-10,,DY096,,GS01SUMO-SBD12-04-PCO2AA000,3,1,2016-11-25T01:11:00,2018-12-09T16:42:00,CGMGS-01SUMO-00003,,CGINS-PCO2AA-34232,-54.4076,-89.35673333,,0,4612,
 NBP16-10,,DY096,,GS01SUMO-SBD12-05-WAVSSA000,3,1,2016-11-25T01:11:00,2018-12-09T16:42:00,CGMGS-01SUMO-00003,,CGINS-WAVSSA-05971,-54.4076,-89.35673333,,0,4612,
-NBP16-10,,DY096,,GS01SUMO-SBD12-06-METBKA000,3,1,2016-11-25T01:11:00,2018-12-09T16:42:00,CGMGS-01SUMO-00003,,CGINS-METLGR-00036,-54.4076,-89.35673333,,-5,4612,
-NBP16-10,,DY096,,GS01SUMO-SBD12-08-FDCHPA000,3,1,2016-11-25T01:11:00,2018-12-09T16:42:00,CGMGS-01SUMO-00003,,CGINS-FDCHPA-43401,-54.4076,-89.35673333,,-5,4612,
-DY096,,DY112,,GS01SUMO-RID16-00-DCLENG000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGCON-NDCL16-00022,-54.407168,-89.206037,,12,4589,
+NBP16-10,,DY096,,GS01SUMO-SBD12-06-METBKA000,3,1,2016-11-25T01:11:00,2018-12-09T16:42:00,CGMGS-01SUMO-00003,,CGINS-METLGR-00036,-54.4076,-89.35673333,,0,4612,
+NBP16-10,,DY096,,GS01SUMO-SBD12-08-FDCHPA000,3,1,2016-11-25T01:11:00,2018-12-09T16:42:00,CGMGS-01SUMO-00003,,CGINS-FDCHPA-43401,-54.4076,-89.35673333,,-6,4612,
```
Comment (snwhite) [inline deployment/GS01SUMO_Deploy.csv:153]: Deployment 3 looks good.

File: deployment/GS01SUMO_Deploy.csv [hunk 4]
```diff
@@ -194,14 +194,14 @@ DY096,,DY112,,GS01SUMO-SBD11-01-MOPAK0000,4,1,2018-12-04T17:10:40,2020-01-20T10:
...
 DY096,,DY112,,GS01SUMO-SBD12-03-HYDGN0000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-HYDGN0-42406,-54.407168,-89.206037,,0,4589,
 DY096,,DY112,,GS01SUMO-SBD12-04-PCO2AA000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-PCO2AA-33183,-54.407168,-89.206037,,0,4589,
 DY096,,DY112,,GS01SUMO-SBD12-05-WAVSSA000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-WAVSSA-05781,-54.407168,-89.206037,,0,4589,
-DY096,,DY112,,GS01SUMO-SBD12-06-METBKA000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-METLGR-00031,-54.407168,-89.206037,,-5,4589,
-DY096,,DY112,,GS01SUMO-SBD12-08-FDCHPA000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-FDCHPA-74102,-54.407168,-89.206037,,-5,4589,
+DY096,,DY112,,GS01SUMO-SBD12-06-METBKA000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-METLGR-00031,-54.407168,-89.206037,,0,4589,
+DY096,,DY112,,GS01SUMO-SBD12-08-FDCHPA000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-FDCHPA-74102,-54.407168,-89.206037,,-6,4589,
```
Comment (snwhite) [inline deployment/GS01SUMO_Deploy.csv:207]: Deployment 4 looks good.

File: deployment/GI01SUMO_Deploy.csv [hunk 5]
```diff
@@ -35,73 +35,73 @@ KN-221-4,,AT-30-01,,GI01SUMO-RII11-02-CTDMOR020,1,1,2014-09-10T18:50:00,2015-08-
...
-AT-30-01,,AR-07-01,,GI01SUMO-SBD12-04-PCO2AA000,2,1,2015-08-15T19:22:00,2016-07-19T00:00:00,CGMGI-01SUMO-00002,,CGINS-PCO2AA-34233,59.94358,-39.57372,,0,2668,
-AT-30-01,,AR-07-01,,GI01SUMO-SBD12-05-WAVSSA000,2,1,2015-08-15T19:22:00,2016-07-19T00:00:00,CGMGI-01SUMO-00002,,CGINS-WAVSSA-05321,59.94358,-39.57372,,0,2668,
-AT-30-01,,AR-07-01,,GI01SUMO-SBD12-06-METBKA000,2,1,2015-08-15T19:22:00,2016-07-19T00:00:00,CGMGI-01SUMO-00002,,CGINS-METLGR-00019,59.94358,-39.57372,,-5,2668,
-AT-30-01,,AR-07-01,,GI01SUMO-SBD12-08-FDCHPA000,2,1,2015-08-15T19:22:00,2016-07-19T00:00:00,CGMGI-01SUMO-00002,,CGINS-FDCHPA-44903,59.94358,-39.57372,,-5,2668,
-AR-07-01,,AR21,,GI01SUMO-RID16-00-DCLENG000,3,1,2016-07-10T17:44:00,2017-08-14T09:33:00,CGMGI-01SUMO-00003,,OL000596,59.9341,-39.4673,,12,2691,
+KN-221-4,,AT-30-01,,GI01SUMO-SBD12-06-METBKA000,1,1,2014-09-10T18:50:00,2015-08-22T11:00:00,CGMGI-01SUMO-00001,,CGINS-METLGR-00005,59.9337,-39.47378,,0,2685,
+KN-221-4,,AT-30-01,,GI01SUMO-SBD12-08-FDCHPA000,1,1,2014-09-10T18:50:00,2015-08-22T11:00:00,CGMGI-01SUMO-00001,,CGINS-FDCHPA-41004,59.9337,-39.47378,,-6,2685,
```
Comment (snwhite) [inline deployment/GI01SUMO_Deploy.csv:53]: Deployment 1 looks good.

File: deployment/GI01SUMO_Deploy.csv [hunk 6]
```diff
@@ -35,73 +35,73 @@ KN-221-4,,AT-30-01,,GI01SUMO-RII11-02-CTDMOR020,1,1,2014-09-10T18:50:00,2015-08-
...
+AT-30-01,,AR-07-01,,GI01SUMO-SBD12-01-OPTAAD000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-OPTAAD-00242,59.94358,-39.57372,,1,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD12-02-FLORTD000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-FLORTD-01296,59.94358,-39.57372,,1,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD12-03-HYDGN0000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-HYDGN0-99999,59.94358,-39.57372,,0,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD12-04-PCO2AA000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-PCO2AA-34233,59.94358,-39.57372,,0,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD12-05-WAVSSA000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-WAVSSA-05321,59.94358,-39.57372,,0,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD12-06-METBKA000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-METLGR-00019,59.94358,-39.57372,,0,2668,
+AT-30-01,,AR-07-01,,GI01SUMO-SBD12-08-FDCHPA000,2,1,2015-08-15T19:22:00,2016-07-19T10:09:00,CGMGI-01SUMO-00002,,CGINS-FDCHPA-44903,59.94358,-39.57372,,-6,2668,
```
Comment (snwhite) [inline deployment/GI01SUMO_Deploy.csv:103]: Deployment 2 CTDMOQ017 depth should be 500 (not 501).

File: deployment/GI01SUMO_Deploy.csv [hunk 7]
```diff
@@ -135,72 +135,72 @@ AR-07-01,,AR21,,GI01SUMO-RII11-02-PCO2WC052,3,1,2016-07-10T17:44:00,2017-08-14T0
...
-AR21,,AR30-03,,GI01SUMO-SBD12-04-PCO2AA000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-PCO2AA-35261,59.9462,-39.4737,,0,2659,buoy disappeared 10/12/17 - not recovered
-AR21,,AR30-03,,GI01SUMO-SBD12-05-WAVSSA000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-WAVSSA-06431,59.9462,-39.4737,,0,2659,buoy disappeared 10/12/17 - not recovered
-AR21,,AR30-03,,GI01SUMO-SBD12-06-METBKA000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-METLGR-00021,59.9462,-39.4737,,-5,2659,buoy disappeared 10/12/17 - not recovered
-AR21,,AR30-03,,GI01SUMO-SBD12-08-FDCHPA000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-FDCHPA-64402,59.9462,-39.4737,,-5,2659,buoy disappeared 10/12/17 - not recovered
+AR-07-01,,AR21,,GI01SUMO-SBD12-06-METBKA000,3,1,2016-07-10T17:44:00,2017-08-14T09:33:00,CGMGI-01SUMO-00003,,CGINS-METLGR-00005,59.9341,-39.4673,,0,2691,
+AR-07-01,,AR21,,GI01SUMO-SBD12-08-FDCHPA000,3,1,2016-07-10T17:44:00,2017-08-14T09:33:00,CGMGI-01SUMO-00003,,CGINS-FDCHPA-33908,59.9341,-39.4673,,-6,2691,
```
Comment (snwhite) [inline deployment/GI01SUMO_Deploy.csv:153]: Deployment 3 looks good.

File: deployment/GI01SUMO_Deploy.csv [hunk 8]
```diff
@@ -135,72 +135,72 @@ AR-07-01,,AR21,,GI01SUMO-RII11-02-PCO2WC052,3,1,2016-07-10T17:44:00,2017-08-14T0
...
+AR21,,AR30-03,,GI01SUMO-SBD12-02-FLORTD000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-FLORTD-01222,59.9449,-39.5751,,1,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD12-03-HYDGN0000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-HYDGN0-42509,59.9449,-39.5751,,0,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD12-04-PCO2AA000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-PCO2AA-35261,59.9449,-39.5751,,0,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD12-05-WAVSSA000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-WAVSSA-06431,59.9449,-39.5751,,0,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD12-06-METBKA000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-METLGR-00021,59.9449,-39.5751,,0,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,AR30-03,,GI01SUMO-SBD12-08-FDCHPA000,4,1,2017-08-05T18:17:00,2018-06-14T08:02:00,CGMGI-01SUMO-00004,,CGINS-FDCHPA-64402,59.9449,-39.5751,,-6,2659,buoy disappeared 10/12/17 - not recovered
```
Comment (snwhite) [inline deployment/GI01SUMO_Deploy.csv:203]: Deployment 4 stopDateTime for all components should be 2017-10-12T09:07:00 (the time the buoy went adrift and the riser fell to the seafloor).  
CUID for Buoy components should be "UNRECOVERED".

File: deployment/GI01SUMO_Deploy.csv [hunk 9]
```diff
@@ -233,71 +233,71 @@ AR30-03,,AR35-05,,GI01SUMO-RII11-02-FLORDG033,5,1,2018-06-08T17:21:00,2019-08-09
...
-AR35-05,,,,GI01SUMO-SBD12-02-FLORTD000,6,1,2019-08-05T15:26:00,,CGMGI-01SUMO-00006,,CGINS-FLORTD-01102,59.945118,-39.57401,,1,2664,
-AR35-05,,,,GI01SUMO-SBD12-03-HYDGN0000,6,1,2019-08-05T15:26:00,,CGMGI-01SUMO-00006,,CGINS-HYDGN0-49876,59.945118,-39.57401,,0,2664,
-AR35-05,,,,GI01SUMO-SBD12-04-PCO2AA000,6,1,2019-08-05T15:26:00,,CGMGI-01SUMO-00006,,CGINS-PCO2AA-35262,59.945118,-39.57401,,0,2664,
-AR35-05,,,,GI01SUMO-SBD12-05-WAVSSA000,6,1,2019-08-05T15:26:00,,CGMGI-01SUMO-00006,,CGINS-WAVSSA-06301,59.945118,-39.57401,,0,2664,
-AR35-05,,,,GI01SUMO-SBD12-06-METBKA000,6,1,2019-08-05T15:26:00,,CGMGI-01SUMO-00006,,CGINS-METLGR-00036,59.945118,-39.57401,,-5,2664,
-AR35-05,,,,GI01SUMO-SBD12-08-FDCHPA000,6,1,2019-08-05T15:26:00,,CGMGI-01SUMO-00006,,CGINS-FDCHPA-64403,59.945118,-39.57401,,-5,2664,
+AR30-03,,AR35-05,,GI01SUMO-SBD12-06-METBKA000,5,1,2018-06-08T17:21:00,2019-08-09T08:04:00,CGMGI-01SUMO-00005,,CGINS-METLGR-00015,59.93295,-39.46549,,0,2693,
+AR30-03,,AR35-05,,GI01SUMO-SBD12-08-FDCHPA000,5,1,2018-06-08T17:21:00,2019-08-09T08:04:00,CGMGI-01SUMO-00005,,CGINS-FDCHPA-71403,59.93295,-39.46549,,-6,2693,
```
Comment (snwhite) [inline deployment/GI01SUMO_Deploy.csv:253]: Deployment 5 looks good.

File: deployment/GI01SUMO_Deploy.csv [hunk 10]
```diff
@@ -233,71 +233,71 @@ AR30-03,,AR35-05,,GI01SUMO-RII11-02-FLORDG033,5,1,2018-06-08T17:21:00,2019-08-09
...
+AR35-05,,,,GI01SUMO-SBD11-08-NUTNRB000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-00006,,CGINS-NUTNRB-01092,59.945118,-39.57401,,1,2664,
+AR35-05,,,,GI01SUMO-SBD12-00-DCLENG000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-00006,,CGCON-BDCL12-50009,59.945118,-39.57401,,0,2664,
+AR35-05,,,,GI01SUMO-SBD12-01-OPTAAD000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-00006,,CGINS-OPTAAD-00187,59.945118,-39.57401,,1,2664,
+AR35-05,,,,GI01SUMO-SBD12-02-FLORTD000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-00006,,CGINS-FLORTD-01102,59.945118,-39.57401,,1,2664,
+AR35-05,,,,GI01SUMO-SBD12-03-HYDGN0000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-00006,,CGINS-HYDGN0-49876,59.945118,-39.57401,,0,2664,
+AR35-05,,,,GI01SUMO-SBD12-04-PCO2AA000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-00006,,CGINS-PCO2AA-35262,59.945118,-39.57401,,0,2664,
+AR35-05,,,,GI01SUMO-SBD12-05-WAVSSA000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-00006,,CGINS-WAVSSA-06301,59.945118,-39.57401,,0,2664,
+AR35-05,,,,GI01SUMO-SBD12-06-METBKA000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-00006,,CGINS-METLGR-00036,59.945118,-39.57401,,0,2664,
+AR35-05,,,,GI01SUMO-SBD12-08-FDCHPA000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-00006,,CGINS-FDCHPA-64403,59.945118,-39.57401,,-6,2664,
```
Comment (snwhite) [inline deployment/GI01SUMO_Deploy.csv:303]: Deployment 6 looks good.

File: deployment/GS01SUMO_Deploy.csv (hunk 2 above)
Comment (rtravis-whoi) [reply deployment/GS01SUMO_Deploy.csv:103]: fixed

File: deployment/GI01SUMO_Deploy.csv (hunk 6 above)
Comment (rtravis-whoi) [reply deployment/GI01SUMO_Deploy.csv:103]: fixed

File: deployment/GI01SUMO_Deploy.csv (hunk 8 above)
Comment (rtravis-whoi) [reply deployment/GI01SUMO_Deploy.csv:203]: fixed recovery cruise ID and stop date/time for deployment 4

File: deployment/GI01SUMO_Deploy.csv (hunk 6 above)
Comment (snwhite) [reply deployment/GI01SUMO_Deploy.csv:103]: Looks good.

File: deployment/GI01SUMO_Deploy.csv [hunk 11]
```diff
@@ -135,72 +135,72 @@ AR-07-01,,AR21,,GI01SUMO-RII11-02-PCO2WC052,3,1,2016-07-10T17:44:00,2017-08-14T0
...
+AR21,,UNRECOVERED,,GI01SUMO-SBD12-02-FLORTD000,4,1,2017-08-05T18:17:00,2017-10-12T09:07:00,CGMGI-01SUMO-00004,,CGINS-FLORTD-01222,59.9449,-39.5751,,1,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,UNRECOVERED,,GI01SUMO-SBD12-03-HYDGN0000,4,1,2017-08-05T18:17:00,2017-10-12T09:07:00,CGMGI-01SUMO-00004,,CGINS-HYDGN0-42509,59.9449,-39.5751,,0,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,UNRECOVERED,,GI01SUMO-SBD12-04-PCO2AA000,4,1,2017-08-05T18:17:00,2017-10-12T09:07:00,CGMGI-01SUMO-00004,,CGINS-PCO2AA-35261,59.9449,-39.5751,,0,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,UNRECOVERED,,GI01SUMO-SBD12-05-WAVSSA000,4,1,2017-08-05T18:17:00,2017-10-12T09:07:00,CGMGI-01SUMO-00004,,CGINS-WAVSSA-06431,59.9449,-39.5751,,0,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,UNRECOVERED,,GI01SUMO-SBD12-06-METBKA000,4,1,2017-08-05T18:17:00,2017-10-12T09:07:00,CGMGI-01SUMO-00004,,CGINS-METLGR-00021,59.9449,-39.5751,,0,2659,buoy disappeared 10/12/17 - not recovered
+AR21,,UNRECOVERED,,GI01SUMO-SBD12-08-FDCHPA000,4,1,2017-08-05T18:17:00,2017-10-12T09:07:00,CGMGI-01SUMO-00004,,CGINS-FDCHPA-64402,59.9449,-39.5751,,-6,2659,buoy disappeared 10/12/17 - not recovered
```
Comment (snwhite) [inline deployment/GI01SUMO_Deploy.csv:203]: Corrections to Deployment 4 look good.

File: deployment/GS01SUMO_Deploy.csv (hunk 2 above)
Comment (snwhite) [reply deployment/GS01SUMO_Deploy.csv:103]: Looks good.

File: deployment/GI01SUMO_Deploy.csv (hunk 11 above)
Comment (dbogorff) [reply deployment/GI01SUMO_Deploy.csv:203]: Deployment 4 looks good!

File: deployment/GI01SUMO_Deploy.csv (hunk 9 above)
Comment (dbogorff) [reply deployment/GI01SUMO_Deploy.csv:253]: Deployment 5 looks good!

File: deployment/GI01SUMO_Deploy.csv (hunk 10 above)
Comment (dbogorff) [reply deployment/GI01SUMO_Deploy.csv:303]: Deployment 6 looks good!

File: deployment/GS01SUMO_Deploy.csv (hunk 1 above)
Comment (dbogorff) [reply deployment/GS01SUMO_Deploy.csv:53]: Deployment 1 looks good!

File: deployment/GS01SUMO_Deploy.csv (hunk 2 above)
Comment (dbogorff) [reply deployment/GS01SUMO_Deploy.csv:103]: Deployment 2 looks good!

File: deployment/GS01SUMO_Deploy.csv (hunk 3 above)
Comment (dbogorff) [reply deployment/GS01SUMO_Deploy.csv:153]: Deployment 3 looks good!

File: deployment/GS01SUMO_Deploy.csv (hunk 4 above)
Comment (dbogorff) [reply deployment/GS01SUMO_Deploy.csv:207]: Deploymnet 4 looks good!

**Pull request merged**